### PR TITLE
Utils/com cleanup

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: pynopegl-utils/pynopegl_utils/export.py [hunk 1]
```diff
@@ -72,15 +72,15 @@ def run(self):
     def _export(self, filename, width, height, extra_enc_args=None):
         fd_r, fd_w = os.pipe()
 
-        cfg = self._get_scene_func()
-        if not cfg:
+        scene_info = self._get_scene_func()
```
[the remark this answers is not in the document]
Comment (ubitux) [reply pynopegl-utils/pynopegl_utils/export.py:84]: `_get_scene_info_func`?

File: pynopegl-utils/pynopegl_utils/export.py [hunk 2]
```diff
@@ -153,7 +153,7 @@ def _get_scene(**cfg_overrides):
         if "error" in ret:
             print(ret["error"])
             return None
-        return ret
+        return ret["ret"]
```
Comment (mbouron) [inline pynopegl-utils/pynopegl_utils/export.py]: nit: ret["ret"] fields a bit weird, would it make sense to rename the local variable `info` ? Same for `odict`.

File: pynopegl-utils/pynopegl_utils/export.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mbouron) [reply pynopegl-utils/pynopegl_utils/export.py:84]: Yes and we can even drop the _func suffix imho.

File: pynopegl-utils/pynopegl_utils/export.py (hunk 2 above)
Comment (ubitux) [reply pynopegl-utils/pynopegl_utils/export.py]: It becomes `query_info.ret` later on, no worry ;)

File: pynopegl-utils/pynopegl_utils/export.py [hunk 3]
```diff
@@ -35,7 +36,15 @@ class Exporter(QtCore.QThread):
...
-    def __init__(self, get_scene_func, filename, w, h, extra_enc_args=None, time=None):
+    def __init__(
+        self,
+        get_scene_func: Callable[..., Optional[SceneInfo]],
+        filename,
+        w,
+        h,
+        extra_enc_args=None,
+        time=None,
+    ):
```
[the remark this answers is not in the document]
Comment (ubitux) [reply pynopegl-utils/pynopegl_utils/export.py:47]: Added a commit on top

**Review by mbouron — APPROVED** [review]

**Pull request merged**